src/commonMain/kotlin/it/krzeminski/snakeyaml/engine/kmp/scanner/Scanner.kt

```diff
@@ -34,6 +34,18 @@ interface Scanner : Iterator<Token> {
      */
     fun checkToken(vararg choices: Token.ID): Boolean
 
+    /**
+     * Check if the next token is the given type.
+     *
+     * @param choice token ID to match with
+     * @return `true` if the next token is the given type. Returns `false`
+     * if no more tokens are available.
+     * @throws ScannerException Thrown in case of malformed input.
+     */
+    fun checkToken(choice: Token.ID): Boolean {
```

Comment:
chore: Please, update the API dump - this is a new method in the public API

Reply:
https://github.com/krzema12/snakeyaml-engine-kmp/pull/302/commits/8f018c02dc6784688a2b50f916f23636488c38d6